keyboards/cipulot/ec_pro2/info.json

```diff
@@ -0,0 +1,184 @@
+{
+    "manufacturer": "Cipulot",
+    "keyboard_name": "EC Pro2",
+    "maintainer": "Cipulot",
+    "board": "GENERIC_STM32_F401XC",
+    "bootloader": "stm32-dfu",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "keyboard_folder": "cipulot/ec_pro2",
+    "mouse_key": {
+        "enabled": true
+    },
+    "processor": "STM32F401",
+    "rgblight": {
+        "led_count": 22,
+        "pin": "B14",
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
+    "url": "https://www.github.com/Cipulot/EC-Pro-2",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x6B8E",
+        "shared_endpoint": {
+            "keyboard": true
+        },
+        "vid": "0x6369"
+    },
+    "layouts": {
+        "LAYOUT_all": {
+            "layout": [
+                { "label": "0,0", "matrix": [0, 0], "w": 1, "x": 0, "y": 0 },
```

Comment:
`"w": 1` is unnecessary, the default key width and height are 1u.

Reply:
@fauxpark done 👍